examples/tutorials/Advanced/Websockets - Federated training/README.md

```diff
@@ -0,0 +1,25 @@
+# Federated learning using websockets - MNIST example
+
+The scripts in this folder let you execute a federated training via three websocket connections.
+
+The script start_websocket_servers.py will start the Websocket server workers for Alice, Bob and Charlie.
+```
+$ python start_websocket_servers.py
+```
+
+The training is then started by running the script run_websocket_client.py:
+```
+$ python run_websocket_client.py
```

Comment:
Any way that we could create a version of this in a notebook so that people trying to use it can iterate step by step?

Reply:
I thought of it, but debugging a notebook is more difficult than putting breakpoints in an editor as pycharm. 
The script starting the websocket servers will not go away, as starting the processes from a notebook does not work (see discussion in the slack channel).
I will see what makes sense in a notebook, as it shouldn't be too long neither.

Reply:
I added a notebook version of the example.